addons/binding/org.openhab.binding.yeelight/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,26 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: yeelight Binding
+Bundle-SymbolicName: org.openhab.binding.yeelight;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.0.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Bundle-ClassPath: .,
+ libs/yeelight_lib.jar
+Import-Package: 
+ com.google.common.collect;version="10.0.1",
```

Comment:
Please remove the [version constraint](http://docs.openhab.org/developers/development/guidelines.html#b-osgi-bundles)